modin/tests/pandas/dataframe/test_map_metadata.py

```diff
@@ -223,13 +223,51 @@ def test_add_suffix(data, axis):
     df_equals(new_modin_df.columns, new_pandas_df.columns)
 
 
+map_strtagies = ["map", "axis_map", "splitted_axis_map"]
+
+
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
 @pytest.mark.parametrize("testfunc", test_func_values, ids=test_func_keys)
 @pytest.mark.parametrize(
     "na_action", [None, "ignore"], ids=["no_na_action", "ignore_na"]
 )
-def test_applymap(data, testfunc, na_action):
+@pytest.mark.parametrize("map_strtagy", map_strtagies, ids=map_strtagies)
+def test_applymap(data, testfunc, na_action, map_strtagy):
```

Comment:
Let's add a test in test_internals.py instead since we want to test lower functionality.

Reply:
Ok, lets do it.